<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirObject</class>
 <widget class="QMainWindow" name="AirObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="minimumSize">
             <size>
              <width>377</width>
              <height>248</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>377</width>
              <height>248</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Загрузить</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Выберите аппарат</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="airBox"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="loadingButton">
                <property name="text">
                 <string>Загрузить</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Создать</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Название аппарата</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nameEdit">
                  <property name="minimumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Длина аппарата</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="longEdit">
                  <property name="minimumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Взлетный вес аппарата</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="weightEdit">
                  <property name="minimumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Скорость полета аппарата</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="speedEdit">
                  <property name="minimumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Картинка</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="photEdit">
                  <property name="minimumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>209</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="loadingPhotButton">
                  <property name="text">
                   <string>Загрузить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="creatButton">
                <property name="text">
                 <string>Создать</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
